build.gradle

```diff
@@ -344,7 +364,7 @@ dependencies {
 
     implementation group: 'uk.gov.hmcts.reform', name: 'logging', version: versions.reformLogging
     implementation group: 'uk.gov.hmcts.reform', name: 'logging-appinsights', version: versions.reformLogging
-    implementation group: 'uk.gov.hmcts.reform', name: 'properties-volume-spring-boot-starter', version: '0.0.4'
+    implementation group: 'uk.gov.hmcts.reform', name: 'properties-volume-spring-boot-starter', version: '0.1.0'
```

Comment:
might need to look into this one as i think its now deprecated

Reply:
this would be a larger ticket as we may need to apply some config stuff for spring, also potential that its already been done and the above is no longer used, I just updated it to match other repos